src/core_plugins/kibana/public/context/api/context.js

```diff
@@ -0,0 +1,52 @@
+import _ from 'lodash';
+
+import { addComputedFields } from './utils/fields';
+import { getDocumentUid } from './utils/ids';
+import { createSuccessorsQuery } from './utils/queries.js';
+import { reverseQuerySort } from './utils/sorting';
+
+
+async function fetchSuccessors(es, indexPattern, anchorDocument, sort, size) {
+  const successorsQuery = prepareQuery(indexPattern, anchorDocument, sort, size);
+  const results = await performQuery(es, indexPattern, successorsQuery);
+  return results;
+}
+
+async function fetchPredecessors(es, indexPattern, anchorDocument, sort, size) {
+  const successorsQuery = prepareQuery(indexPattern, anchorDocument, sort, size);
+  const predecessorsQuery = reverseQuerySort(successorsQuery);
+  const reversedResults = await performQuery(es, indexPattern, predecessorsQuery);
```

Comment:
OK, sounds good! I think this makes more sense to me now and we can ignore my original comment. Thanks for the explanation.